src/Raven.Client/Documents/Session/AsyncDocumentSession.Stream.cs

```diff
@@ -152,24 +153,21 @@ public Task<IAsyncEnumerator<StreamResult<T>>> StreamAsync<T>(IQueryable<T> quer
             return StreamAsync(indexQuery, token);
         }
 
-        public Task<IAsyncEnumerator<StreamResult<T>>> StreamAsync<T>(long? fromEtag, int start = 0,
-                                                                    int pageSize = Int32.MaxValue, string transformer = null, Dictionary<string, object> transformerParameters = null, CancellationToken token = default(CancellationToken))
+        public Task<IAsyncEnumerator<StreamResult<T>>> StreamAsync<T>(string fromChangeVector, int start = 0, int pageSize = int.MaxValue, string transformer = null, Dictionary<string, object> transformerParameters = null, CancellationToken token = default(CancellationToken))
```

Comment:
Order is a funny thing in a distributed env.
We can return all the results after a particular change vector, but different servers will return them in different orders.
That can be expensive and require us to scan the whole db, though.